docs/pages/index.mdx

```diff
@@ -1,25 +1,72 @@
+The vision of this proposal is to standardize how smart contracts send and receive messages via a common interface, regardless of the messaging protocol employed. 
 
-## How It Works
+This unlocks improved crosschain user experience, mitigates risk, and enables a new class of crosschain native smart contracts while improving Ethereum interoperability by enabling composability among the various crosschain messaging protocols. 
 
-- Sending Messages: A SourceGateway contract allows a sender to specify the destination chain, receiver address, message payload, and optional attributes. The gateway processes the request and emits a MessagePosted event.
-- Receiving Messages: A DestinationGateway validates the message using the cross-chain protocol and delivers it to the receiver contract via the executeMessage function.
-- Attributes: Attributes enable advanced functionality such as setting gas limits, passing metadata, or handling protocol-specific requirements.
+## Introduction to Crosshain Messaging
 
-## Example Use Cases
+Ethereum’s crosschain journey has been driven by its wide adoption success. With a vibrant community of builders introducing new use cases, markets, and ideas, the requirements of the builders and DApps grew. 
+
+The need to scale was evident and the Ethereum community answered the call. While the expansion of the L2 network has been widely successful in bringing down transaction costs, enabling experimentation, and facilitating the entry of new users to the Ethereum ecosystem, it has not been without challenges of its own. One of these challenges has been interoperability. 
+
+Different blockchains, such as Ethereum and the network of L2s and rollups, do not natively communicate with each other. 
+
+Crosschain messaging protocols enable interoperability between these different protocols by allowing smart contracts on one blockchain to interact with contracts and assets on another through the transfer of data. 
+
+One popular use case of messaging protocols is bridges. These are often used to transfer assets, tokens, and NFTs across blockchain networks for use in DeFi protocols, on-chain games, and socialFi.
+
+There are currently a number of protocols which enable crosschain communication. While this growing network of protocols has made crosschain activities a reality, the development of multiple different technical solutions and methodologies has come at the cost of broader compostability and a unified developer and user experience. 
+
+## ERC-7786 at a Glance
+
+The objective of ERC-7786 is to provide a standard interface to send and receive crosschain messages between smart contracts. 
+
+
+It allows app developers to use a common framework, minimizing lock-in to specific crosschain messaging technologies. Protocol-specific features can remain available under this common framework through non-standard attributes, without breaking interoperability of the core functionality of simple cross-chain messaging.
```

Comment:
```suggestion
It allows app developers to use a common framework, minimizing lock-in to specific crosschain messaging technologies. Protocol-specific features can remain available under this common framework through extensible attributes, without breaking interoperability of the core functionality of simple cross-chain messaging.
```

Attributes can be standardized so I wouldn't call them "non-standard" in general.